internal/config/config.go

```diff
@@ -38,6 +38,7 @@ type Config struct {
 	Name             string        `default:"nsc" desc:"Name of Network Service Client"`
 	ConnectTo        url.URL       `default:"unix:///var/lib/networkservicemesh/nsm.io.sock" desc:"url to connect to NSM" split_words:"true"`
 	ConnectTimeout   time.Duration `default:"5s" desc:"timeout to connect to NSMgr" split_words:"true"`
+	RequestTimeout   time.Duration `default:"5m" desc:"timeout to request NSE" split_words:"true"`
```

Comment:
Why 5 min?

Reply:
5 min is not too long and not too short :)

Reply:
Reduced to 15s.

Reply:
It is long